### PR TITLE
fix: Postgresql flexible server - pre-commit issue

### PR DESCRIPTION
**Pull request by fmlisco**
### Type of changes

- [ ] Add new module
- [x] Update existing module
- [ ] Remove existing module

### Does this introduce a breaking change?

- [ ] Yes
- [x] No

### Run checks

Useful commands to run checks on local machine

```sh
bash .utils/terraform_run_all.sh init local
pre-commit run -a
```

### CODE REVIEW

File: postgres_flexible_server/tests/resources.tf [hunk 1]
```diff
@@ -118,7 +118,10 @@ module "postgres_flexible_server_private" {
   log_analytics_workspace_id                = azurerm_log_analytics_workspace.test.id
   diagnostic_setting_destination_storage_id = module.storage_account.id
 
-  depends_on = [azurerm_private_dns_zone_virtual_network_link.privatelink_postgres_database_azure_com_vnet]
+  #####
+  ##### Questo comando fa fallire pre-commit nella Github Action
```
Comment (uolter) [inline postgres_flexible_server/tests/resources.tf:122]: english pls

**Review by re-sh-cloud-arch — DISMISSED** [review]

File: postgres_flexible_server/tests/resources.tf [hunk 2]
```diff
@@ -137,8 +140,11 @@ resource "azurerm_key_vault_secret" "pgres_flex_admin_pwd" {
   value        = random_password.password.result
   key_vault_id = module.key_vault_test.id
   content_type = "password"
-
-  depends_on = [azurerm_key_vault_access_policy.pgsql, azurerm_key_vault_access_policy.user]
+    
+  #####
+  ##### Questo comando fa fallire pre-commit nella Github Action
```
Comment (uolter) [inline postgres_flexible_server/tests/resources.tf:145]: english pls

**Review by uolter — CHANGES_REQUESTED** [review]

File: postgres_flexible_server/tests/resources.tf (hunk 1 above)
Comment (fmlisco) [reply postgres_flexible_server/tests/resources.tf]: done

File: postgres_flexible_server/tests/resources.tf (hunk 2 above)
Comment (fmlisco) [reply postgres_flexible_server/tests/resources.tf]: done

**Review by uolter — DISMISSED** [review]

**Pull request merged**